commitizen/commands/commit.py

```diff
@@ -61,7 +61,9 @@ def prompt_commit_questions(self) -> str:
 
         if not answers:
             raise NoAnswersError()
-        return cz.message(answers)
+
+        message_length_limit: int = self.arguments.get("message_length_limit", 0)
+        return cz.message(answers, message_length_limit=message_length_limit)
```

Comment:
One interesting subtlety here: there is a reason why `_check_message_length_limit` is not called here, but instead called in `message` of each child of `BaseCommitizen`.
It's currently because for conventional commits the whole message may consist of multiple lines, where the limit should only be applied to the first line.
There is a possible alternative: call `_check_message_length_limit` and only take stuff before the first line break (`\n`).
But I assume the current form allows more flexible settings for each child of `BaseCommitizen`, so did not adopt that alternative.

Reply:
Yep, I think we should take care of only the first line. and this behavior should be clearly documentated

Reply:
No problem.
The new commit simplifies accordingly, and the behavior is also reflected in commit.md.
A missing test `test_commit_command_with_message_length_limit` is also added.